src/app/Console/Commands/Install.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Backpack\Base\app\Console\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+
+class Install extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'backpack:base:install';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Require dev packages and publish files for Backpack\Base to work';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->info("### Backpack\Base installation started. Please wait...");
+
+        $this->executeProcess('composer require backpack/generators --dev');
+        $this->executeProcess('composer require laracasts/generators:dev-master --dev');
```

Comment:
Should the above two be made optional (but default to true)? Technically Backpack will work without them.

Reply:
Yup, you're right. They could be optional. I thought including them in require-dev will do no harm (they won't be present in production), but it should make it more intuitive for the user: "_of course the generators are included automagically on dev_". You think it's confusing rather than helpful?

Reply:
The more that is required, the more likely something will break - it's up to you.